cmd/testgrid-config-generator/main.go

```diff
@@ -270,6 +254,31 @@ func main() {
 		logrus.WithError(err).Fatal("Could not process input configurations.")
 	}
 
+	// read the list of jobs from the allow list along with its release-type
+	var allowList map[string]string
+	if o.jobsAllowListFile != "" {
+		data, err := ioutil.ReadFile(o.jobsAllowListFile)
+		if err != nil {
+			logrus.WithError(err).Fatalf("could not read allow-list at %s", o.jobsAllowListFile)
+		}
+		allowList, err = getAllowList(data)
+		if err != nil {
+			logrus.WithError(err).Fatalf("failed to build allow-list dictionary")
+		}
```

Comment:
I think the initial reasoning behind keeping this bit before the validation function was so that we could get away with not passing the some parameters (the testgrid directory I think) if we were running it in the validationOnly mode. Why would we not need that any longer?